requirements.txt

```diff
@@ -0,0 +1,6 @@
+unipath>=1.1,<=2.0
+django>=1.8,<=1.9
+git+https://github.com/rosskarchner/django-sheerlike.git#egg=sheerlike
```

Comment:
Just until the PR's get pulled in-- there's no code in the CFPB version of sheerlike yet.